frontend/src/expert/expertGlobals.js

```diff
@@ -186,8 +227,17 @@ export const MIST_builtin_values = {
         "name": "m.y",
         "about": "takes the y-value of the position of the mouse on the image"
     },
+    "rad": {
```

Comment:
These don't seem to actually draw anything, as far as we can tell. Are they used elsewhere? Are they defined? What are they supposed to do?

Reply:
I'm pretty sure this is what feeds into the information that gets displayed in the tooltips. I added these to accommodate for the functions I'd added. You guys can dismiss things having to do with functions, since they are preserved in the FunctionAdditions branch. That goes for all of them, I think.